kafka-connect-http/src/main/java/com/github/castorm/kafka/connect/http/response/timestamp/RegexTimestampParserConfig.java

```diff
@@ -0,0 +1,54 @@
+package com.github.castorm.kafka.connect.http.response.timestamp;
+
+/*-
+ * #%L
+ * kafka-connect-http
+ * %%
+ * Copyright (C) 2020 CastorM
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import com.github.castorm.kafka.connect.http.response.timestamp.spi.TimestampParser;
+import lombok.Getter;
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+
+import java.util.Map;
+
+import static org.apache.kafka.common.config.ConfigDef.Importance.HIGH;
+import static org.apache.kafka.common.config.ConfigDef.Importance.LOW;
+import static org.apache.kafka.common.config.ConfigDef.Type.CLASS;
+import static org.apache.kafka.common.config.ConfigDef.Type.STRING;
+
+@Getter
+public class RegexTimestampParserConfig extends AbstractConfig {
+  private static final String ITEM_TIMESTAMP_REGEX = "http.response.record.timestamp.parser.regex";
+  private static final String PARSER_DELEGATE = "http.response.record.timestamp.parser.delegate";
```

Comment:
Since this is becoming a pattern, if we chain multiple parsers, we'll need delegates to be declared under different property names, otherwise only one delegate would be possible.

So I'd suggest changing this property name to make it specific of this parser.
```java
"http.response.record.timestamp.parser.regex.delegate"
```